fs/copy.ts

```diff
@@ -0,0 +1,206 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as path from "./path/mod.ts";
+import { ensureDir, ensureDirSync } from "./ensure_dir.ts";
+
+export interface CopyOptions {
+  /* overwrite existing file or directory, default is false */
+  overwrite?: boolean;
+}
+
+function isSrcSubdir(src: string, dest: string): boolean {
```

Comment:
there are two `isSrcSubdir` method in the fs modules

Should it be extracted into `path` modules?

Reply:
What does it do? Please add JSDoc.

And yes, we should not duplicate functionality.